### DISCUSSION TITLE
Error in idx[i, ] <- res[[i]][[1]]  and Number of anchors is less than k.weight

### DISCUSSION THREAD

**Discussion by sylestiel** [stuart-lab/signac#1367] [General]
Hi Tim,

Despite numerous attempts at integration of 2 separate Multiome datasets I keep running into the following error when attempting to run IntegrateEmbeddings():

Error in idx[i, ] <- res[[i]][[1]] :
number of items to replace is not a multiple of replacement length
In addition: Warning message:
In RunIntegration(filtered.anchors = filtered.anchors, normalization.method = normalization.method, :
Number of anchors is less than k.weight. Lowering k.weight for sample pair.

I can't seem to get past this.  Any help would be much appreciated.

**Comment by timoast** [comment]
Hi, please open an issue in the Seurat github page as this function is part of Seurat
